src/main/java/Nell.java

```diff
@@ -0,0 +1,217 @@
+import java.util.Scanner;
+
+public class Nell {
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+
+    /**
+     * Prints out the formatted string for the task at a specified index
+     *
+     * @param task The task at the specified index
+     * @param index The index of task
+     */
+    private static void printTaskAtIndex(Task task, int index) {
+        System.out.println(String.format("   %d. %s", index, task));
+    }
+
+    /**
+     * Given a task number, checks if the numbered task is valid (in the task list)
+     *
+     * @param taskNumber the task number
+     * @return true if taskIndex is within range, false otherwise
+     */
+    private static boolean checkIfTaskValid(int taskNumber) {
+        if (taskNumber < 1) {
+            return false;
+        } else if (taskNumber > taskCount) {
+            return false;
+        } else {
+            return true;
+        }
+    }
+
+    /**
+     * Adds a new task to the task list
+     *
+     * @param taskToAdd The task to be added to the list
+     */
+    private static void listAddTask(Task taskToAdd) {
+        // Stores text in task list
+        tasks[taskCount] = taskToAdd;
+        taskCount++;
+    }
+
+    /**
+     * Lists out the currently stored tasks in TaskList, upon receipt of the
+     * list command
+     *
+     */
+    private static void executeCommandList() {
+        // List out stored tasks
+        System.out.println("-> The tasks listed are as follows:");
+        for (int i = 0; i < taskCount; i++) {
+            // Prints all tasks in list
+            printTaskAtIndex(tasks[i], (i + 1));
+        }
+    }
+
+    /**
+     * Executes a todo command with a given command body
+     *
+     * @param commandBody The command body
+     */
+    private static void executeCommandToDo(String commandBody) {
+        System.out.println("-> The task has been added to the list:");
+        ToDo toDoToAdd = new ToDo(commandBody);
+        listAddTask(toDoToAdd);
+        System.out.println("   " + toDoToAdd);
+        System.out.println(String.format("   The list now has %d tasks", taskCount));
+    }
+
+    /**
+     * Executes a deadline command with a given command body
+     *
+     * @param commandBody The command body
+     */
+    private static void executeCommandDeadline(String commandBody) {
+        System.out.println("-> The task has been added to the list:");
+        String[] commandWords = commandBody.split("/by");
+        Deadline deadlineToAdd = new Deadline(commandWords[0].trim(), commandWords[1].trim());
+        listAddTask(deadlineToAdd);
+        System.out.println("   " + deadlineToAdd);
+        System.out.println(String.format("   The list now has %d tasks", taskCount));
+    }
+
+    /**
+     * Executes an event command with a given command body
+     *
+     * @param commandBody The command body
+     */
+    private static void executeCommandEvent(String commandBody) {
```

Comment:
commandBody sounds a bit clunky and confusing. It might be better to call it "description" for clarity.

Reply:
Thanks for your comment, I agree with your suggestion.